suite-common/toast-notifications/src/notificationsSelectors.ts

```diff
@@ -47,3 +49,13 @@ export const selectOpenedTransactionNotifications = (state: NotificationsRootSta
             (n.type === 'tx-received' || n.type === 'tx-sent' || n.type === 'tx-confirmed'),
     ) as TransactionNotification[];
 };
+
+export const selectNativeToastNotifications = (state: NotificationsRootState) => {
```

Comment:
In common only code related to both platforms should be present, otherwise move this to some native specific package.